closed/OpenJ9.gmk

```diff
@@ -96,11 +96,14 @@ else
 endif
 
 ifeq ($(OPENJDK_TARGET_OS),windows)
+  # set Visual Studio environment
+  EXPORT_MSVS_ENV_VARS := PATH="$(PATH)" INCLUDE=$(INCLUDE) LIB=$(LIB)
```

Comment:
The values of INCLUDE and LIB will also likely need to be enclosed in double-quotes.

Reply:
On second reading, I see those variables get their quotes from custom-spec.gmk. A comment noting that difference would be helpful.

Reply:
I added comments